device/rocev2/flowsettings.yaml

```diff
@@ -0,0 +1,213 @@
+      description: >-
+            This configuration allows you to configure RDMA flow over the same QP number from same source and destination.
+      type: object
+      properties:
+        connection_type:
+          description: >-
+            There are multiple connection types. Valid values are :  Reliable Connection (RC), Reliable Datagram (RD), Extended Reliable Connection (XRC),
+            Unreliable Datagram (UD),  Unreliable Connection (UC), Raw IPv6 Datagram, Raw Ethertype Datagram.
+          type: string
+          x-enum:
+            reliable_connection:
+              x-field-uid: 1
+            reliable_datagram:
+              x-field-uid: 2
+            extended_reliable_connection:
+              x-field-uid: 3
+            unreliable_datagram:
+              x-field-uid: 4
+            unreliable_connection:
+              x-field-uid: 5
+            raw_ipv6_datagram:
+              x-field-uid: 6
+            raw_ethernet_datagram:
+              x-field-uid: 7
+          x-field-uid: 1
+        source_qp_number:
+          description: >-
+            Configure the QP range.
+          type: integer
+          format: uint32
+          default: 2
+          minimum: 2
+          maximum: 33554431
+          x-field-uid: 2
+        dscp:
+          description: >-
+            DSCP value for this flow
+          type: integer
+          format: uint32
+          default: 24
+          minimum: 0
+          maximum: 63
+          x-field-uid: 3
+        ecn:
+          description: >-
+            This field allows to configure bits of the Traffic Class field in the IPv4 or IPv6 header to encode four different code points.
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 0
+          maximum: 3
+          x-field-uid: 4
+        udp_source_port:
+          description: >-
+            UDP source port number for this flow.
+          type: integer
+          format: uint32
+          default: 49152
+          minimum: 0
+          maximum: 65535
+          x-field-uid: 5
+        rocev2_verb:
+          description: >-
+            RoCEv2 Verb, Available options are: none, write, wrtie_with_immediate, send, send_with_immediate and read: The corresponding flow will not take part in traffic.
+          type: string
+          default: write
+          x-enum:
+            none:
+              x-field-uid: 1
+            write:
+              x-field-uid: 2
+            write_with_immediate:
+              x-field-uid: 3
+            send:
+              x-field-uid: 4
+            send_with_immediate:
+              x-field-uid: 5
+            read:
+              x-field-uid: 6
+          x-field-uid: 6
+        immediate_data:
+          description: >-
+            Immediate Data field required for SEND/WRITE with immediate verb.
+          type: string
+          format: hex
+          default: "00000000"
+          x-field-uid: 7
+        message_size:
```

Comment:
Is message_size part of session bring up or is it needed solely for data flows e.g. could a single QP potentially first send a payload of size 1000 bytes and then another of size 10000 bytes ? In which probably should be somewhere in stateful_flow configuration as properties of the data flow , maybe as an array with ixnetwork supporting a single value only for all flows between same qp endpoints. Or is this specification bound that each qp can have single datagram being split and sent on it ?